### PR TITLE
Add Stork for indexed search.

### PR DESCRIPTION
**Pull request by leerob**
Still need to run `stork --build` somewhere in the build process, but it's mostly working locally.

https://github.com/jameslittle230/stork

### CODE REVIEW

**Comment by leerob** [comment]
#### 📝Changed routes:


| [**/blog**](https://leerob-git-stork-search.leerob1.now.sh/blog) |
|:-:|
|<a href="https://leerob-git-stork-search.leerob1.now.sh/blog">Screenshot of /blog</a><br /><sup><a href="https://images.weserv.nl?url=https://s.microlink.io/Kk7vMYUNbFdI1lZHTGZ_auGVBu0_.png">(view full size)</a> |







Commit 54917eb5f411a24a8e3c81957bba2e3938622d95 (https://leerob-git-stork-search.leerob1.now.sh).

File: scripts/index-search.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const fs = require('fs');
+const path = require('path');
+const matter = require('gray-matter');
+
+const root = process.cwd();
+const files = fs.readdirSync(path.join(root, 'data', 'blog'));
+
+const blogPosts = files.reduce((allPosts, slug) => {
```
Comment (CodeByCorey) [inline scripts/index-search.js:8]: I was curious why you chose to use reduce over map here.

Comment (leerob) [reply scripts/index-search.js:8]: Because I copy-pasted this from another file 😆 Good suggestion for `map`!

**Comment by leerob** [comment]
I'm going to revisit this later 👋

**Comment by DavidWells** [comment]
I tried a bunch of different search tools and landed on https://github.com/roxiness/poindexter

Demo of this over in https://vendia.net/docs/share. Not totally finished with the experience/interactions but it works great and can't beat the free pricetag of static search 😃 

(Storks demo https://stork-search.net/ doesn't work for some reason in my chrome ¯\_(ツ)_/¯)

**Comment by leerob** [comment]
@DavidWells Nice, thanks for sharing that!

**Pull request closed without merging**